file-formats/aifp/examples/z_aff_example_aifp.aifp.json

```diff
@@ -0,0 +1,29 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Example AIFP for ABAP File Format",
+    "originalLanguage": "en",
+    "abapLanguageVersion": "cloudDevelopment"
+  },
+  "generalInformation": {
+    "namespace": "AIFNS",
+    "aifCheck": "BP_EXIST"
+  },
+  "singleCheckAssignment": [
+    {
+      "number": "10",
+      "description": "Check Business Partner Exist",
+      "scenario": "simpledb",
+      "fieldCheck": {
+        "fieldCheck": "empty",
```

Comment:
there are two elements "fieldCheck" directly after each other.
 "fieldCheck": {
        "fieldCheck": "empty",

Is there maybe a better name for the inner fieldCheck?

Reply:
@huber-nicolas, thanks for the hint. I changed the name and description for the inner fieldCheck to checkType, which fits better to the purpose of the field.